.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -13,6 +13,11 @@ If this PR depends on other PRs, please list them below this comment,
 using the following format:
 - [ ] depends on: #abc [optional extra text]
 - [ ] depends on: #xyz [optional extra text]
+
+To indicate co-authors, include lines at the bottom of the commit message 
+using the format:
+
+Co-authored-by: Author Name <author@email.com>
```

Comment:
This placement is a little confusing; `Co-authored-by:` has to come _before_ the `---`, but is described after `depends on` which goes _after_ the `---`.

Reply:
Changed.